index.js

```diff
@@ -0,0 +1,9 @@
+function strictEqual(firstValue, secondValue) {
+  // eslint-disable-next-line eqeqeq
+  if (firstValue == secondValue && typeof (firstValue) == typeof (secondValue)) {
+    return true
+  } else {
+    return false
+  }
+}
```

Comment:
This is a great solution. Just something in the future to possibly make your life easier; if there is only one if statement in the function, you can make it a one liner. As follows
```
 function strictEqual(val1, val2) {
  return val1 == val2 && typeof val1 == typeof val2
}
```